src/extensions/UserDefinedFormControllerExtension.php

```diff
@@ -285,4 +288,138 @@ public function getShareLink()
             return $partialSubmission->getPartialLink();
         }
     }
+
+    /**
+     * When form is submitted finished() function is called in
+     * vendor/silverstripe/userforms/code/Control/UserDefinedFormController.php
+     *
+     * SubmissionSummary is added to $data (Pass by reference) which is shown in template
+     * vendor/silverstripe/userforms/templates/SilverStripe/UserForms/Control/UserDefinedFormController_ReceivedFormSubmission.ss
+     * this template is overridden in MBIE site
+     */
+    public function updateReceivedFormSubmissionData(&$data)
+    {
+        if ($this->owner->ShowSubmissionSummary) {
+            $formID = $this->owner->ID;
+            $submissionID = $this->owner->request
```

Comment:
Recommend to use public methods instead of public properties
```suggestion
            $submissionID = $this->owner->getRequest()
```

Reply:
On it.